distros/spark/remote.list

```diff
@@ -5,10 +5,9 @@ twitter,https://mobile.twitter.com/cache/twitter.webapp
 sechelt,http://sechelt.fxosapps.org/manifest.webapp
 yammer,http://yammer.fxosapps.org/manifest.webapp
 hexgl,http://hexgl.fxosapps.org/manifest.webapp
-foxfooding,https://fxos.github.io/foxfooding/manifest.webapp
```

Comment:
We're cutting the Foxfooding app, and instead doing email updates to foxfooders.